Pages/ManageUserEvents.razor

```diff
@@ -50,3 +43,10 @@
         }
     }
 </div>
+
+<MatSelect Label="Page size" Value="@perPage" ValueChanged="async(int a) => await Refresh(a)">
+    <MatOption TValue="int" Value="3">3</MatOption>
+    <MatOption TValue="int" Value="5">5</MatOption>
+    <MatOption TValue="int" Value="10">10</MatOption>
+</MatSelect>
+<Pagination TotaPagesQuantity="m_totalPageQuantity" CurrentPage="m_currentPage" Radius="2" SelectedPage="SelectedPage"></Pagination>
```

Comment:
Maybe I just missed but when searching for an event in index page it works perfectly, but on manage volunteer page it doesn't.

Reply:
Yep, my bad, forgot to do the same changes on ManageUserEvents. It should work properly now.